### PR TITLE
chore(contributors): auto update

### PR DESCRIPTION
**Pull request by sarbbottam**
Updating contributors w.r.t. `.all-contributorsrc` is a mechanical activity, I was wondering if it can be automated.

Feel free to disagree, however if you feel it's usable please share your thoughts.

@ta2edchimp @kentcdodds @jfmengels

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +24,6 @@ after_success:
 branches:
   only:
     - master
+env:
+  global:
+  - GH_REF: github.com/ta2edchimp/opt-cli
```
Comment (sarbbottam) [inline .travis.yml:29]: `GH_TOKEN` needs to be added, please refer [Encrypted-Variables](https://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables) for details.

**Comment by kentcdodds** [comment]
Looks super cool if it works! :+1:

File: script/update-contributors.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+GH_TOKEN="GH_TOKEN"
+GH_REF="GH_REF"
+if [[ $(env 2> /dev/null | grep "$GH_TOKEN" | cut -d'=' -f1) != "$GH_TOKEN" ]] &&
+  [[ $(env 2> /dev/null | grep "$GH_REF" | cut -d'=' -f1) != "$GH_REF" ]]; then
+  exit 0
+fi
+if [[ $(git status 2> /dev/null | grep README | cut -d' ' -f4) == "$README" ]]; then
+  ( git add "$README"
+    git commit -m "docs(readme): updated contributors" -n
+    git push --quiet "https://${GH_TOKEN}@${GH_REF}" master:gh-pages > /dev/null 2>&1
```
Comment (sarbbottam) [inline script/update-contributors.sh:15]: typo, should be `master` no `:gh-pages`

**Comment by sarbbottam** [comment]
I would follow up on this, [having problem in signing into travis from cli](https://twitter.com/sarbbottam/status/713189292751736833).

**Comment by ta2edchimp** [comment]
Thanks for this! :+1: 

At the moment I'll stick to manually invoking the `generate` command (or asking contributors to do it after adding themselves to `.all-contributorsrc`, or using the `add` command in the first place) for **this** project.

But this is totally something I am interested in and it could be super useful for larger projects!
If it doesn't get merged into `all-contributors-cli` as an option, you should definitely turn it into a complementary tool.

**Comment by ta2edchimp** [comment]
:exclamation: **Please do not close this Pull Request!** (I may change my mind) :exclamation:

**Comment by sarbbottam** [comment]
this should be done by `all-contributors`

**Pull request closed without merging**